### ISSUE TITLE
inline handlers ignored on dynamically created elements

### ISSUE THREAD

**Issue by rkernahan** [clientside/amplesdk#46]
It seems that inline handlers are only effective on original markup.
If an element is created dynamically, it dos no good to add an onclick attribute before appending it to the ample tree.
Instead, an event listener needs to be added.

Is this the desired behavior? or should inline handlers work for dynamically inserted elements?

**Comment by clientside** [comment]
Handlers assignement at runtime with 
element.setAttribute("onxxx", "function(e){alert(e)}") 
is not implemented and hence doesn't work. 
Handlers assigned at runtime with:
element.onxxx = function(e){alert(e)} 
will work.

Do you think assigning handlers with attributes should work? (it is doable, but somehow I think it is weird to assign functions as strings - you loose ability to include execution context etc.) Tell me.

**Comment by rkernahan** [comment]
I agree that it would be weird to add an attribute to an element _already_ in the tree and expect the handler to work, but that is not the case I raised.

I meant to say that I'm creating elements to append to a document fragment. Sometime between createElement and appendChild it seems reasonable to add attributes to the newly created element.
I actually did expect that if I add an onclick attribute to an element before it is appended to a document fragment, and before it goes into the DOM, that it would work like markup. 
But it appears not to in my test page.
I'll send the code by reply email.
